### PR TITLE
Adds several missing cases to tuple shrinking.

### PR DESCRIPTION
**Pull request by jacob-tock**
the tuple\* functions are intended to shrink all combinations of fields
in a tuple, but several cases were missing. This PR adds them in.

### CODE REVIEW

**Comment by mgold** [comment]
:+1: LGTM

File: src/Shrink.elm [hunk 1]
```diff
@@ -298,6 +298,7 @@ tuple4 (shrinkA, shrinkB, shrinkC, shrinkD) (a, b, c, d) =
   +++ Lazy.List.map2 (\a b   -> (a,b,c,d)) (shrinkA a) (shrinkB b)
   +++ Lazy.List.map3 (\b c d -> (a,b,c,d)) (shrinkB b) (shrinkC c) (shrinkD d)
   +++ Lazy.List.map3 (\a c d -> (a,b,c,d)) (shrinkA a) (shrinkC c) (shrinkD d)
+  +++ Lazy.Lest.map3 (\a b d -> (a,b,c,d)) (shrinkA a) (shrinkB b) (shrinkD d)
```
Comment (robertjlooby) [inline src/Shrink.elm]: `Lest`, spelling error here

Comment (mgold) [inline src/Shrink.elm]: Uh-oh. That should have been caught by the compiler, right? If OP didn't even compile before submitting, that's not good.

Comment (robertjlooby) [inline src/Shrink.elm]: That's what I was thinking too

:scream_cat: :scream_cat: :scream_cat:

Comment (jacob-tock) [inline src/Shrink.elm]: I did, but it looks like I did the "run the compiler in the wrong directory" trick :(

Anyway, I can push a fix.

Comment (jacob-tock) [inline src/Shrink.elm]: Pushed.

**Comment by yoni-tock** [comment]
Merge?

**Comment by mgold** [comment]
NRI seem to have better things to do than watch these repos (which perhaps should be elm-community` but they got here first). But they gave me write access...

**Pull request merged**